generate.sh

```diff
@@ -233,19 +233,22 @@ type Error int
 const (
 EOF
 
-grep "(ER_" ${SRC_ERRORS} | \
-	# Remove comments symbols.
-	sed "s/\/\*//" | \
-	sed "s/\*\/_(//" | \
-	# Remove and of values.
-	sed "s/) *\\\//" | \
-	# Remove comma at an end of a value.
-	sed "s/,/ /" | \
-	# Finally print parsed values in Golang format.
-	awk '{
+# Extract body of define ERROR_CODES
+ERROR_CODES=`sed -n '/^#define ERROR_CODES(/,/This one should be last/{//!p;}' ${SRC_ERRORS}`
+
+echo "${ERROR_CODES}" | \
+	# Get line with ER_* description
+	sed -nE 's/^[^E]*(ER_.+)\s*\\\s*$/\1/p'  | \
+	# Remove trailing parenthesis or comment's ending
+	sed -E 's/(.*)\s*(:?\)|\*\/)\s*$/\1/' | \
+	awk '
```

Comment:
Please, comment each line. It is really hard to understand what happened here after a few month. Comments may help.
```suggestion
	sed -E 's/(.*)\s*(:?\)|\*\/)\s*$/\1/' | \
	# Foo
	awk '
```